libs/map/src/components/legend/vl-map-legend.ts

```diff
@@ -252,9 +258,11 @@ export class VlMapLegend extends BaseLitElement {
         }
 
         return html` <div class="uig-map-legend" style="${this.generateItemStyle()}">
-            <div>
-                <span class="uig-map-legend-text uig-map-legend-title">Legende: </span>
-            </div>
+            ${this.hideTitle
+                ? nothing
```

Comment:
Ik zou hier een lege string plaatsen ipv `nothing`.
`nothing` wordt gebruikt om een attribuut weg te laten indien er geen waarde is.